bundles/k3d-slim-dev/uds-bundle.yaml

```diff
@@ -55,6 +55,24 @@ packages:
             - name: TENANT_TLS_KEY
               description: "The TLS key for the tenant gateway (must be base64 encoded)"
               path: tls.key
+        gateway:
+          variables:
+            - name: TENANT_SERVICE_PORTS
+              description: "The ports that are exposed from the tenant gateway LoadBalancer (useful for non-HTTP(S) traffic)"
+              path: "service.ports"
+              default:
+                - name: status-port
+                  port: 15021
+                  protocol: TCP
+                  targetPort: 15021
+                - name: http2
+                  port: 80
+                  protocol: TCP
+                  targetPort: 80
+                - name: https
+                  port: 443
+                  protocol: TCP
+                  targetPort: 443
```

Comment:
On the fence on the default - while it is a nice example, it's also something extra to maintain/keep in parity with the upstream defaults. Looking upstream it doesn't look like the ports have changed since the chart was created 3+ years ago, so this might be okay.

Reply:
Removed it - my initial fear was that it would replace with an empty list but tested that and it does not - I think if people are asking for how to do this we could just point to GL as an example.